src/components/admonition/admonition.tsx

```diff
@@ -0,0 +1,124 @@
+import { Component, h, Prop } from '@stencil/core';
+import { AdmonitionType } from '@limetech/lime-elements';
+
+/**
+ * Admonitions, also known as call-outs, very useful for including supportive content
+ * inside a user interface, or within a large piece of text.
+ *
+ * When used in a document or text based user interface, the admonitions attract
+ * reader's attention to a particular piece of information,
+ * without significantly interrupting their flow of reading the document.
+ *
+ * On a user interface, their usage will be experienced more intrusively by
+ * end-users. But they could be good choices when you indent to slightly
+ * disturb users' attention, and challenge them to pay extra attention
+ * to the information presented. In such cases, admonitions should not
+ * be used as static and constantly present elements of a UI. They should
+ * rather be displayed when something unusual or remarkable demands
+ * users attention.
+ *
+ * @exampleComponent limel-example-admonition-info
+ * @exampleComponent limel-example-admonition-warning
+ * @exampleComponent limel-example-admonition-danger
+ * @exampleComponent limel-example-admonition-note
+ * @exampleComponent limel-example-admonition-tip
+ * @exampleComponent limel-example-admonition-success
+ * @exampleComponent limel-example-admonition-failure
+ * @exampleComponent limel-example-admonition-example
+ * @exampleComponent limel-example-admonition-quote
+ * @exampleComponent limel-example-admonition-question
+ * @exampleComponent limel-example-admonition-custom-heading
+ * @exampleComponent limel-example-admonition-styles
+ * @exampleComponent limel-example-admonition-composite
+ */
+@Component({
+    tag: 'limel-admonition',
+    shadow: true,
+    styleUrl: 'admonition.scss',
+})
+export class Admonition {
+    /**
+     * Title of the admonition
+     */
+    @Prop({ reflect: true })
+    public heading?: string;
+
+    /**
+     * Description of the admonition
+     */
+    @Prop({ reflect: true })
+    public description?: string;
```

Comment:
I think we should call this `content`. This is because:
1. if we add markdown support, that'd be a more logical choice of name
2. it works better with `heading` (I had to change the `title` to `heading`, due to build warnings which I commented on earlier)

Reply:
I think this should probably be removed entirely, and all the content should be moved to the child elements slot instead. That means the consumer can format the content however they please, e.g.

```html
<limel-admonition>
    This text is <strong>bold</strong>
</limel-admonition>
```

It also means we can add stuff like markdown support later, e.g.
```html
<limel-admonition>
    <limel-markdown>
        # Lorem ipsum
    </limel-markdown>
</limel-admonition>
```

Reply:
Great idea! I'll fix up

Reply:
@jgroth but what if they wanna use this component in let's say a start page or overview tab of the CRM, and wanna quickly add some content as a string?

Can we just keep this prop, as is (maybe rename it to `content`), and then add a `<slot name="content" />` as well, for those who want to add complex content?

Reply:
In theory it would of course be possible to do that, but the API would perhaps feel a bit weird and conflicting.
How should this scenario be handled?
```html
<limel-admonition content="Is this my content?">
    Maybe this is my content?
</limel-admonition>
```

Instead, perhaps it should be possible to configure children when configuring components in lime-admin and then it would be easy to do something like that. But that's another topic on its own :smile:

Reply:
Well, the case you show is a bit strange. I don't think the user will do so. :) 
If they do, the component will then show them 👇 
![image](https://user-images.githubusercontent.com/35954987/227530003-0e9a5865-6bb6-4252-bb9b-b55dbb159282.png)


I think this discussion is a little bit like another discussion we had regarding `limel-tooltip`. We wanted to enable a `slot` for that component as well, so that consumers can put custom and complex content in it. 

I think it's great that the component gives basic functionality out of the box, but still leaves space for consumers, to help them handle complexities like this. 

This way, they can more easily make something like this: 
![image](https://user-images.githubusercontent.com/35954987/227531132-d23c5b48-f804-445c-8dad-790bfc6552f3.png)

Reply:
honestly, I don't know what you mean 😓 but maybe you will see what I mean, when I add my fixup 😅

Reply:
My point is that we should not add both a prop and allow slotted content if they are meant to be used for the same thing, especially not if the only purpose is to make it easier to configure in lime-admin. If there is a need to be able to configure child elements/content in lime-admin, we should add support for that rather than designing an API for this component that is hard to understand.

Reply:
You're right. I understand. Usage is Lime Admin was just an example. That is not why I was suggesting this approach.

Again, I want to refer to the discussion we had to the tooltip. You remember? We wanted to display hotkey visualizations (like: <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>) inside the tooltip. 

Sometimes, what's offered out of the box is not enough for the customer, and we never know what is enough or can't really support it fully. This is specially the case for components that are heavily dependent on displaying content, description, or rich-text. This component is one of those (which is why the Markdown discussions started).

So, hopefully, my last fixups and example will explain what I mean. :)

Reply:
@Kiarokh I think your example in lime-admin could be done using the property `innerText` or `innerHTML`

Reply:
how do you mean @TommyLindh2 ? Can you add an example code? Or some screenshot?

Reply:
> @Kiarokh I think your example in lime-admin could be done using the property `innerText` or `innerHTML`

`innerText` would work, but it feels a bit hackish :smile: 
`innerHtml` should almost never be used, and here it would render any HTML as text instead of creating nested tags

Reply:
I realized that you cannot use `innerHTML` and `innerText` on custom components, so ignore my comment.
But you cannot add a lime element directly in the view-config anyways, so you'll need a wrapper component, and then you can add your own content property. 

I don't think this will be an issue.